### PR TITLE
Add examples of invoking contracts using Python SDK and Java SDK.

### PR DESCRIPTION
**Pull request by overcat**
For easier review, I suggest starting with the first commit.

### CODE REVIEW

File: docs/smart-contracts/guides/transactions/invoke-contract-tx-js.mdx [hunk 1]
```diff
@@ -44,9 +58,18 @@ This is a simple example using the `@stellar/stellar-sdk` JavaScript library to
     networkPassphrase: Networks.FUTURENET,
   })
     // The invocation of the `increment` function of our contract is added
-    // to the transaction. Note: `increment` doesn't require any parameters,
-    // but many contract functions do. You would need to provide those here.
-    .addOperation(contract.call("increment"))
+    // to the transaction.
+    .addOperation(
```
Comment (overcat) [inline docs/smart-contracts/guides/transactions/invoke-contract-tx-js.mdx:62]: `increment` needs parameters.

File: docs/smart-contracts/guides/transactions/invoke-contract-tx-sdk.mdx [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+
+<TabItem value="py" label="Python">
+
+:::tip
+
+Please go to [the project homepage](https://github.com/StellarCN/py-stellar-base) of Python SDK to learn how to install it.
+
+In addition, Python provides a lot of example code, which you can find [here](https://github.com/StellarCN/py-stellar-base/tree/main/examples).
+
+:::
```
Comment (Julian-dev28) [inline docs/smart-contracts/guides/transactions/invoke-contract-tx-sdk.mdx:148]:
```
ledger=15219 create_at=1718219004
Transaction result: 7
```

Nicely done

File: docs/smart-contracts/guides/transactions/invoke-contract-tx-sdk.mdx [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+<Tabs>
+<TabItem value="js" label="JavaScript">
+
+:::tip
+
+Please go to [the project homepage](https://github.com/stellar/js-stellar-sdk) of JavaScript SDK to learn how to install it.
+
+:::
+
+```javascript
```
Comment (Julian-dev28) [inline docs/smart-contracts/guides/transactions/invoke-contract-tx-sdk.mdx:20]: Having some issues with this one. 
```
Sending transaction failed
{"_attributes":{"feeCharged":{"_value":"94770"},"result":{"_switch":{"name":"txBadAuth","value":-6}},"ext":{"_switch":0}}}
```

Are you able to add me on discord? my username is thegodhand_

**Review by Julian-dev28 — APPROVED** [review]
Nicely done!

**Pull request merged**